source/_integrations/reolink.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: Reolink NVR/camera
+description: Instructions on how to integrate Reolink devices (NVR/cameras) into Home Assistant.
+ha_category:
+  - Camera
+ha_iot_class: Local Polling
+ha_release: 2023.1
+ha_domain: reolink
+ha_codeowners:
+  - '@starkillerOG'
+ha_config_flow: true
+ha_platforms:
+  - Camera
+ha_integration_type: integration
+---
+
+The integration allows you to control [Reolink](https://reolink.com/) NVRs or cameras.
+
+{% include integrations/config_flow.md %}
+
+## Known reolink firmware bugs
+
+<div class='note warning'>
+There is a BUG in Reolink NVR firmware: it only sends ONVIF event-notifications if motion happened on the camera connected to its very first (index "0") channel.
+Reolink is aware of this and are working on fixing the problem.
+</div>
+
+<div class='note warning'>
+Reolink doorbell presses only generate ONVIF event-notifications when the doorbell is directly connected to your network.
+When connecting the Reolink doorbell to a NVR, no doorbell press events are generated.
+Reolink is aware of this and a solution is already implemented in beta firmware.
+</div>
+
+## Camera streams
+
+This integration creates a few camera entities, one for each stream type with different resolutions: Main, Sub, Ext and Images.
+The Sub stream camera entity is enabled by default, the other streams are disabled by default.
+The Images stream provides a sequence of image snapshots giving very low latency at the cost of a very low frame rate, this can be used when the hi-res RTMP/RTSP video stream are too laggy.
```

Comment:
```suggestion
The Images stream provides a sequence of image snapshots giving very low latency at the cost of a very low frame rate; this can be used when the hi-res RTMP/RTSP video stream has too much lag.
```